src/GitHub.Exports/Helpers/ICopyable.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace GitHub.Helpers
+{
+    public interface ICopyable<in T>
```

Comment:
Please document the interface and the method. What's its purpose, etc. I didn't see any implementations of this interface. I assume that comes later? :smile: